### ISSUE TITLE
Start reaching out to universities about GitHub Education

### ISSUE THREAD

**Issue by froi** [Code4PuertoRico/Code4PR#10]
GitHub has some really good resources for universities, professors, and students. We should start reaching out to education about [GitHub Education and the resources that are available](https://education.github.com/). 

There are a number of resources and support for educators and students alike. The experiences I've seen from others in LatAm with this program have been incredible.

The impact in PR would be huge, especially with the constant issues of students not knowing how to collaborate on software projects, version control, and let alone GitHub.

## Who to contact

- UPR Rio Piedras
- UPR Mayagüez
- UPR Bayamon
- Politecnica
- Engine-4

### Universities, Bootcamp, and Schools

- https://education.github.com/schools

#### TLDR

As a GitHub Education school, you'll enjoy:

- Free access to GitHub Enterprise Server and GitHub Enterprise Cloud for every department, including your technical departments
- Teacher training to master Git and GitHub with our Campus Advisors program
- Automated access to premium GitHub Education features, like the GitHub Student Developer Pack
- Exclusive access to new features, and GitHub Education-specific swag

### Teachers

- https://education.github.com/teachers

#### TLDR

Teachers get access to free enterprise grade tools to help make them successful in the classroom.

### Students

- https://education.github.com/students

#### TLDR

Students will get access to free tools, domains, and swag to start off their tech/dev careers on track. They also have access to the Campus Expert program that gives them leadership opportunities in their school community. This is backed up by mentorship from GitHub and the GitHub Ed. community.

**Comment by froi** [comment]
Talked to the GitHub Education team and they are willing to clear up any question I might have before reaching out to the institutions.

At the same time some community members have reached out saying they are willing to help make this a reality for their schools. 🎉

**Comment by froi** [comment]
This is still on our radar. Now that we have a core team I can take this up and push it over the finish line.
